src/ai/backend/client/request.py

```diff
@@ -267,6 +268,10 @@ def _sign(
                 pass
         else:
             raise ValueError("unsupported endpoint type")
+        bai_ip = self.session.aiohttp_session.headers.get("X-BackendAI-IP") or socket.gethostbyname(
+            socket.gethostname()
+        )
+        self.headers.update({"X-BackendAI-IP": bai_ip})
```

Comment:
As manager can extract client's IP address automatically (via request.remote and `X-Forwarded-For` header) I think it's useless to bundle arbitrary header representing originated IP address.
```suggestion
        bai_ip = self.session.aiohttp_session.headers.get("X-BackendAI-IP") or socket.gethostbyname(
            socket.gethostname()
        )
        self.headers.update({"X-BackendAI-IP": bai_ip})
```